docs/api/structures/shared-texture-import-texture-info.md

```diff
@@ -0,0 +1,8 @@
+# SharedTextureImportTextureInfo Object
+
+* `pixelFormat` string - The pixel format of the texture.
```

Comment:
Is there a list of expected values for pixel format? Maybe we could provide a string enum for this.

Reply:
Good point, added.